src/main/java/org/osiam/resources/controller/ResourceController.java

```diff
@@ -64,4 +74,29 @@
         result.add("id");
         return result;
     }
+
+    protected MappingJacksonValue buildWrapper(T resource, String attributes) {
```

Comment:
Why do you use two `buildWrapper` at all? Why not using 

`protected MappingJacksonValue buildWrapper(Object resource, String attributes) { ... }` ?